src/map/MapSidebar.js

```diff
@@ -310,6 +311,24 @@ function MapSidebarMain (props) {
     }
   }, [api, cid, participantCount]);
 
+  const [nextMeetupTime, setNextMeetupTime] = useState([]);
+  // gets the date of the next Meetup
+  useEffect(() => {
+    async function getNextMeetupDate () {
+      const tilebelt = require('@mapbox/tilebelt');
+      const timespace = require('@mapbox/timespace');
+      const meetupLocations = await api.rpc.encointer.getLocations(cid);
+      const tempLocation = locationFromJson(api, meetupLocations[0]);
```

Comment:
most likely, the sign gets lost somewhere here. It looks like Green Bay longitude is evaluated as positive